src/tribler/core/components/restapi/restapi_component.py

```diff
@@ -43,6 +43,7 @@
 class RESTComponent(Component):
     rest_manager: RESTManager = None
     root_endpoint: RootEndpoint = None
+    swagger_mode: bool = False
```

Comment:
`swagger_mode` seems unclear to me. By swagger, the following UI comes to mind where users can test out REST APIs. By having this variable as default false, it could be assumed that the UI might not be available as well, which is not the case. I'd suggest renaming it something in line with extraction related.
![swagger-ui-tribler](https://user-images.githubusercontent.com/1442867/177500041-aac2c1c4-7c02-403f-852b-dd3523669cd2.png)

Reply:
@xoriole what name would you prefer?

Reply:
Maybe `swagger_doc_extraction_mode`?

Reply:
Alexander's suggestion sounds good

Reply:
Done